### PR TITLE
Project dashboard

### PR DESCRIPTION
**Pull request by devton**
- Created templates for project status on dashboard-home

### CODE REVIEW

File: app/assets/stylesheets/catarse_bootstrap/_extra.sass [hunk 1]
```diff
@@ -98,3 +98,11 @@ select.date
 
 .hidden
   display: none
+
+.text-color-for-status
```
Comment (diogob) [inline app/assets/stylesheets/catarse_bootstrap/_extra.sass]: We should talk to @thiagocatarse and include it in the webflow

**Comment by diogob** [comment]
Let's move the strings to the i18n files.

File: app/assets/stylesheets/catarse_bootstrap/_extra.sass (hunk 1 above)
Comment (thiagocatarse) [inline app/assets/stylesheets/catarse_bootstrap/_extra.sass]: yep @diogob actually these classes were already there

rejected & failed                            -->     .text-error
waiting_funds                                -->     .text-waiting
approved, online e succesfull       -->      .text-success

maybe @devton didn't know about that

**Comment by devton** [comment]
@diogob @adrianob this PR is not ready yet :P

File: app/assets/javascripts/app/project/edit.js [hunk 2]
```diff
@@ -29,3 +35,40 @@ App.addChild('ProjectEdit', _.extend({
   }
 }, Skull.Tabs));
 
+// FIXME: MORE DRY HERE !!
```
Comment (devton) [inline app/assets/javascripts/app/project/edit.js]: @diogob @adrianob i need some help here :/ I don't wanna repeat this code some suggestions?

Comment (diogob) [inline app/assets/javascripts/app/project/edit.js]: I believe we should use the ProjectEdit for everything. Just call all the this.route

Comment (devton) [inline app/assets/javascripts/app/project/edit.js]: @diogob the problem is the data-action generated when we use render :edit :/
or i just cant change the el to a wrapper class?

Comment (diogob) [inline app/assets/javascripts/app/project/edit.js]: Let's use another wrapper element that you can add to the edit template and use the same for all these actions ;)

**Pull request merged**